pkgs/by-name/pa/passwdqc/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, fetchzip
+, libxcrypt
+, linux-pam
+, stdenv
+}:
+stdenv.mkDerivation rec {
+  pname = "passwdqc";
+  version = "2.0.3";
+
+  src = fetchzip {
+    url = "https://www.openwall.com/passwdqc/passwdqc-${version}.tar.gz";
+    hash = "sha256-EgPeccqS+DDDMBVMc4bd70EMnXFuyglftxuqoaYHwNY=";
+  };
+
+  buildInputs = [
+    linux-pam
+    libxcrypt
+  ];
+
+  makeFlags = [
+    "BINDIR=${placeholder "out"}/bin"
+    "DEVEL_LIBDIR=${placeholder "out"}/lib"
+    "SHARED_LIBDIR=${placeholder "out"}/lib"
+    "SECUREDIR=${placeholder "out"}/lib/secure"
+    "INCLUDEDIR=${placeholder "out"}/include"
+    "MANDIR=${placeholder "out"}/share/man"
+    "LOCALEDIR=${placeholder "out"}/share/locale"
+    "CONFDIR=${placeholder "out"}/etc"
+  ];
+
+  meta = with lib; {
+    description = "Password/passphrase strength checking and enforcement";
+    homepage = "https://www.openwall.com/passwdqc/";
+    platforms = platforms.linux;
+    license = licenses.bsd0;
```

This doesn't seem entirely correct

https://www.openwall.com/passwdqc/LICENSE

Parts of the code seems to be `bsd3` (and gpl2, but that's not included in the tarball?)

```suggestion
    license = with licenses; [ bsd0 bsd3 ];
```